addons/website/views/website_navbar_templates.xml

```diff
@@ -109,9 +109,46 @@
-                            <ul>
-                                <li groups="website.group_website_designer"><a href="#" data-action="new_page" aria-label="New page" title="New page"><i class="fa fa-file-o"/><p>New Page</p></a></li>
-                            </ul>
+                            <div class="container pt32 pb32">
+                                <div class="row">
+                                    <div groups="website.group_website_designer" class="col-md-4 mb8 o_new_content_element">
+                                        <a href="#" data-action="new_page" aria-label="New page" title="New page">
+                                            <i class="fa fa-file-o"/>
+                                            <p>New Page</p>
+                                        </a>
+                                    </div>
+                                    <div groups="base.group_system" name="module_website_blog" t-att-data-module-id="env.ref('base.module_website_blog').id" t-att-data-module-shortdesc="env.ref('base.module_website_blog').shortdesc" class="col-md-4 mb8 o_new_content_element">
+                                        <a href="#" data-action="new_blog_post">
+                                            <i class="fa fa-rss"/>
+                                            <p>New Blog Post</p>
+                                        </a>
+                                    </div>
+                                    <div groups="base.group_system" name="module_website_event" t-att-data-module-id="env.ref('base.module_website_event').id" t-att-data-module-shortdesc="env.ref('base.module_website_event').shortdesc" class="col-md-4 mb8 o_new_content_element">
+                                        <a href="#" data-action="new_event">
+                                            <i class="fa fa-glass"/>
+                                            <p>New Event</p>
+                                        </a>
+                                    </div>
+                                    <div groups="base.group_system" name="module_website_forum" t-att-data-module-id="env.ref('base.module_website_forum').id" t-att-data-module-shortdesc="env.ref('base.module_website_forum').shortdesc" class="col-md-4 mb8 o_new_content_element">
+                                        <a href="#" data-action="new_forum">
+                                            <i class="fa fa-comment"/>
+                                            <p>New Forum</p>
+                                        </a>
+                                    </div>
+                                    <div groups="base.group_system" name="module_website_hr_recruitment" t-att-data-module-id="env.ref('base.module_website_hr_recruitment').id" t-att-data-module-shortdesc="env.ref('base.module_website_hr_recruitment').shortdesc" class="col-md-4 mb8 o_new_content_element">
+                                        <a href="/jobs/add">
```

Comment:
@seb-odoo shouldn't it be `href="#"`?

Reply:
why would you like to have smthing that do nothing ? 

Most of other object uses click binding to create their new record. $('.new_forum').on('click', fun())
In case of job, the controller /job/add do it without js, so a direct link seems legits.

Do you have a bug ?

Reply:
@FalcoBolger It was question of changing it to # in website but with an override in website_job to the correct url (because the url doesn't exist in website itself, there was an issue with an url crawler or something like that).

But for the feature itself as @JKE-be said, the other actions are handled in JS but this one is just a link so it should be different in that regard.

Reply:
@JKE-be @seb-odoo the crawler fails when the website_hr_recruitment module is not installed, so it seems better to have the module itself adding the link to the endpoint.

Just a suggestion, nothing that an inherited template could not fix.

Reply:
See PR #29217